usaspending_api/disaster/management/sql/disaster_covid19_file_b.sql

```diff
@@ -30,9 +30,36 @@ SELECT
+    "financial_accounts_by_program_activity_object_class"."gross_outlays_undelivered_orders_prepaid_total_cpe" AS "gross_outlays_undelivered_orders_prepaid_total_FYB",
+    "financial_accounts_by_program_activity_object_class"."gross_outlays_delivered_orders_paid_total_cpe" AS "USSGL480200_undelivered_orders_obligations_prepaid_advanced",
+    "financial_accounts_by_program_activity_object_class"."gross_outlays_delivered_orders_paid_total_fyb" AS "USSGL480200_undelivered_orders_obligations_prepaid_advanced_FYB",
+    "financial_accounts_by_program_activity_object_class"."ussgl488200_up_adjust_pri_undeliv_order_oblig_ppaid_adv_cpe" AS "USSGL488200_upward_adj_prior_year_undeliv_orders_oblig_prepaid",
+    "financial_accounts_by_program_activity_object_class"."gross_outlays_delivered_orders_paid_total_cpe" AS "gross_outlays_delivered_orders_paid_total",
+    "financial_accounts_by_program_activity_object_class"."gross_outlays_delivered_orders_paid_total_fyb" AS "gross_outlays_delivered_orders_paid_total_FYB",
+    "financial_accounts_by_program_activity_object_class"."ussgl490200_delivered_orders_obligations_paid_cpe" AS "USSGL490200_delivered_orders_obligations_paid",
+    "financial_accounts_by_program_activity_object_class"."ussgl490800_authority_outlayed_not_yet_disbursed_cpe" AS "USSGL490800_authority_outlayed_not_yet_disbursed",
+    "financial_accounts_by_program_activity_object_class"."ussgl490800_authority_outlayed_not_yet_disbursed_fyb" AS "USSGL490800_authority_outlayed_not_yet_disbursed_FYB",
+    "financial_accounts_by_program_activity_object_class"."ussgl498200_upward_adjust_pri_deliv_orders_oblig_paid_cpe" AS "USSGL498200_upward_adj_of_prior_year_deliv_orders_oblig_paid",
     "financial_accounts_by_program_activity_object_class"."deobligations_recoveries_refund_pri_program_object_class_cpe" AS "deobligations_or_recoveries_or_refunds_from_prior_year",
-    "financial_accounts_by_program_activity_object_class"."ussgl487200_down_adj_pri_ppaid_undel_orders_oblig_refund_cpe" AS "downward_adj_prior_yr_ppaid_undeliv_orders_oblig_refunds_cpe",
-    "financial_accounts_by_program_activity_object_class"."ussgl497200_down_adj_pri_paid_deliv_orders_oblig_refund_cpe" AS "downward_adj_prior_yr_paid_delivered_orders_oblig_refunds_cpe",
+    "financial_accounts_by_program_activity_object_class"."ussgl487100_down_adj_pri_unpaid_undel_orders_oblig_recov_cpe" AS "USSGL487100_downward_adj_prior_year_unpaid_undeliv_orders_oblig",
+    "financial_accounts_by_program_activity_object_class"."ussgl497100_down_adj_pri_unpaid_deliv_orders_oblig_recov_cpe" AS "USSGL497100_downward_adj_prior_year_unpaid_deliv_orders_oblig",
+    "financial_accounts_by_program_activity_object_class"."ussgl487200_down_adj_pri_ppaid_undel_orders_oblig_refund_cpe" AS "USSGL487200_downward_adj_prior_year_prepaid_undeliv_order_oblig",
```

Comment:
Is this title (`USSGL487200_downward_adj_prior_year_prepaid_undeliv_order_oblig`) getting truncated in the download CSV? We ran into this before after we had worked on the COVID downloads because there is a max character limit for an annotation of 63 characters (or some number close to that).

Reply:
Nope, that column name is exactly 63 characters

Reply:
Oh sweet, I miscounted earlier.